azblob/zt_examples_test.go

```diff
@@ -1,4 +1,4 @@
-package azblob_test
+package azblob
 
```

Comment:
This is a test file and (generally) should not be in the azblob package since these were designed to test the convenience layer. Since there's a large enough impact on these zt_* files, could you point to the places that made this change necessary? Alternatively, you could move the necessary bits to a separate lower level test file.